blocks/job-list/job-list.css

```diff
@@ -0,0 +1,63 @@
+.job-list {
+  --color-text: var(--spectrum-gray-900);
+  --color-border: var(--spectrum-gray-400);
+  margin-block: 2rem;
+}
+
+.job-list__department {
```

Comment:
can we can we rename these modifiers to something generic, less related to the content? For example, instead of "department," maybe "sub-heading" or "detail"?